libqtile/backend/x11/window.py

```diff
@@ -892,21 +886,232 @@ def place(
         self.width = width
         self.height = height
 
-        kwarg = dict(
+        kwargs = dict(
             x=x,
             y=y,
             width=width,
             height=height,
         )
         if above:
-            kwarg["stackmode"] = StackMode.Above
-
-        self.window.configure(**kwarg)
+            self.change_layer(kwargs=kwargs)
+        else:
+            self.window.configure(**kwargs)
```

Comment:
There is confusing overlap in the role of `self.change_layer` and `self.window.configure` and I think we should segregate those two things. This would also remove `kwargs` from `change_layer` which AFAICT is an x11-specific detail.
```suggestion
        self.window.configure(
            x=x,
            y=y,
            width=width,
            height=height,
	)
        if above:
            self.change_layer(up=True)
```
```

Reply:
Yes. Much neater.